config.go

```diff
@@ -250,6 +252,17 @@ func ParseConfig(fn string) (Configuration, error) {
 		}
 	}
 
+	highlights_by_docs, ok := input.Get("bridge", "highlights_by_doc")
```

Comment:
A few things here:

We want to make this more configurable, by listing keys that we want to eliminate from the output (it could be `highlights_by_doc` today, but also `really_annoying_key` tomorrow as well)

So make this configurable with an option to list the keys to remove from the output, for example:

`remove_from_output=highlights_by_doc,actiontype`

- that would remove both the `actiontype` and `highlights_by_doc` fields in any output that we generate. Of course the default would be to not remove anything (exactly what we're doing now)